From 2bc94cc3bb99aa3d1ad7898f02d89ca61ca31966 Mon Sep 17 00:00:00 2001
From: hscham <hscham@chromium.org>
Date: Mon, 25 Oct 2021 14:13:46 +0900
Subject: [PATCH] Add base/containers/cxx20_erase.h stub header

crrev.com/c/2970616 (r896181) moved base::to_address and
base::Erase{,If} to separate files.

Change-Id: Ifada289b40a969ec0f773122a58d32f994f08438
---
 base/containers/cxx20_erase.h | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 base/containers/cxx20_erase.h

diff --git a/base/containers/cxx20_erase.h b/base/containers/cxx20_erase.h
new file mode 100644
index 0000000000..7dedc8a832
--- /dev/null
+++ base/containers/cxx20_erase.h
@@ -0,0 +1,6 @@
+#ifndef BASE_CONTAINERS_CXX20_ERASE_H_
+#define BASE_CONTAINERS_CXX20_ERASE_H_
+
+#include "base/stl_util.h"
+
+#endif  // BASE_CONTAINERS_CXX20_ERASE_H_
-- 
2.33.0.1079.g6e70778dc9-goog

